<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="mysql-test-runner" tests="3" failures="1" skipped="0" time="0.138" timestamp="2025-06-17T09:49:25.333159+00:00">
  <properties>
    <property name="os" value="macos"/>
    <property name="rust_version" value="0.2.0"/>
    <property name="git_commit" value="5dbafa9840b8db4703f036eea0ea65011e940250"/>
    <property name="cli_args" value="target/debug/dingo_test_runner simple_test echo_test error_test --xunit-file demo_report.xml"/>
  </properties>
  <testcase name="simple_test" classname="mysql-test.simple_test" time="0.056"/>
  <testcase name="echo_test" classname="mysql-test.echo_test" time="0.035"/>
  <testcase name="error_test" classname="mysql-test.error_test" time="0.047">
    <failure message="Test failed" type="TestFailure">
<![CDATA[Query 5 failed: MySqlError { ERROR 1364 (HY000): Field 'id' doesn't have a default value }
Result verification failed: Output missing lines starting at expected line 3:
    Expected: INSERT INTO test_table (name) VALUES ('Bob')]]>
    </failure>
  </testcase>
</testsuite>
